tests/unit/user_management/user_registry/test_verification_codes.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Unit tests for verification code generation."""
+
+from pytest import mark
+
+from auth_service.user_management.user_registry.core.verification_codes import (
+    generate_code,
+    hash_code,
+    validate_code,
+)
+
+
+def test_generate_verification_code_with_default_size():
+    """Test the generation of verification codes with default size."""
+    codes = set()
+    for _ in range(100):
+        code = generate_code()
+        codes.add(code)
+        assert len(code) == 6
```

Comment:
Depends a bit on whether you consider that constant part of the API or an internal detail.

I tend to not import such constants since if I rename them in the implementation, then the test unnecessarily breaks. And if I set it to the wrong value in the implementation, the test would not detect it.